ui/design-system-components/src/components/Entry/EntryBox/index.tsx

```diff
@@ -212,10 +216,10 @@ const EntryBox: React.FC<IEntryBoxProps> = props => {
 
   return (
     <>
-      <div className={tw(`${error && 'bg-[#FFFDF1]'}`)} style={style}>
-        <div className={tw(`flex flex-row justify-between p-4 shrink-0`)}>
-          <a
-            className={tw(`flex min-w-0 no-underline`)}
+      <Box customStyle={`${error && 'bg-[#FFFDF1]'}`} style={style}>
```

Comment:
don't we have this color defined?

Reply:
I'll take a look now and also fix other hard-coded colors in that file.